docs/user-guide/docs-source/index.md

```diff
@@ -1,37 +1,13 @@
 
-AWS Ambit Scenario Designer for Unreal Engine 4 (Ambit) is a suite of tools to streamline content creation at scale for autonomous vehicle and robotics simulation applications.
+AWS Ambit Scenario Designer for Unreal Engine 4 (Ambit) is a suite of tools to streamline 3D content creation at scale for autonomous vehicle and robotics simulation applications. From procedural placement of obstacles to automatic generation of whole cities, Ambit can help you quickly create the 3D environments and scenarios you need for your simulations or other 3D applications.
 
-Currently Ambit can be installed as a plugin for [Unreal Engine 4.27](https://docs.unrealengine.com/en-US/Support/Builds/ReleaseNotes/4_27/index.html).
+![Ambit sample image](images/AmbitBanner.jpg)
 
-## Getting Started
+Ambit is implemented as an Unreal Engine plugin and has been tested for compatibility with [Unreal Engine 4.27](https://docs.unrealengine.com/en-US/Support/Builds/ReleaseNotes/4_27/index.html).
 
-You can install the plugin either directly in the engine folder (in "Engine/Plugins/Editor/Ambit") or in your project folder (in "Plugins/Editor/Ambit").
 
-Make sure the Ambit plugin is installed and enabled in your Unreal Engine installation by going to **Edit** -> **Plugins** and ensuring the **enabled** checkbox for Ambit is checked.
 
-![Plugins Dropdown](./images/home/PluginDropdown.png)
-
-![Plugin Menu Enabled](./images/home/AmbitPlugin.png)
-
-## AWS Ambit Scenario Mode
-
-Once the plugin is enabled a new **AWS Ambit Scenario Mode** will be added to the **Modes** dropdown menu in the top panel. Select this Mode to enable scenario generation for your scene.
-
-![Modes Dropdown](./images/home/EnableAmbitMode_426.png)
-
-## Enabling Content
-
-Once the Ambit plugin is installed and enabled you will need to enabled plugin content in your content browser to access some of Ambit's tools. To do this, go to the **View Options** dropdown in the bottom right hand corner of your content browser and select **"Show Plugin Content"**.
-
-![Plugin Content Menu Enabled](./images/home/EnablePluginContent.png)
-
-Then "Ambit Content" folder should be selected in the "Sources Panel" of the Content Browser.
-
-![Source Panel Enabled](./images/home/SourcePanel.png)
-
-## Note
-
-During the usage of this plugin tool, Ambit may collect generic user metrics to inform us about the usage of this plugin tool. These metrics contain no identifiable info and are not shared or transferred.
-
-If you prefer to opt out from this metrics collection, within the Ambit plugin folder, you can edit the C++ header file Source/Ambit/Utils/UserMetricsSubsystem.h to make the RecordUserMetrics field false.
+> ✏️ **Note:** During the usage of this plugin tool Ambit will collect generic metrics to inform AWS about usage patterns and performance of its individual features. These metrics contain no personally identifiable information and are not shared or transferred.
+>
```

Comment:
When loading up the website, this generates an empty white line between the two, which looks awkward. Consider removing.

Reply:
rukekre mentioned this in a separate comment. I won't be fixing this at this time. See my [full reply to rukekre's comment](https://github.com/aws-samples/aws-ambit-scenario-designer-ue4/pull/3#discussion_r760615445) for details.